pkg/util/tpm/tpm.go

```diff
@@ -0,0 +1,433 @@
+package tpm
```

Comment:
Basically all of the Validate* functions in this package should have tests. I don't even really know what to look for, and would be terrified of modifying this without tests.